### PR TITLE
WIP - Split files for easier development

### PR DESCRIPTION
**Pull request by hakuch**
Please see the individual commits.

@espindola, @senior7515: What do you think? I know this was something you both expressed interest in.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,14 @@ To understand how to use `cmake-cooking` with your project, see [APPLY.md](./APP
 
 The only file necessary for using `cmake-cooking` in your C or C++ project is [`cooking.sh`](./cooking.sh).
 
+On development branches, this file has to be generated:
+
+```
+$ make cooking.sh
+```
+
+On release branches (and `master`), this file should be present at the top-level of the repository.
```
Comment (hakuch) [inline README.md]: This should say "On tagged releases".

**Comment by espindola** [comment]
I would even leave cooking.sh out of git completely. It is similar to how sometimes configure.ac but not configure is in git, but that is a project by project decision.

In any case, my editor will be very happy that there is now a .cmake file :-)

**Comment by hakuch** [comment]
@espindola: That was the intention (`git rm cooking.sh`), but something got lost in the shuffle. :-)

**Comment by senior7515** [comment]
much easier to read :) +1 !

**Comment by hakuch** [comment]
Thanks for the feedback!

**Pull request closed without merging**